src/main/scripts/src/app/shared/organization.ts

```diff
@@ -0,0 +1,13 @@
+import {Project} from './project';
+
+export class Organization {
```

Comment:
I would suggest moving this class and `Project` class to `app/shared/dto` folder since there will be many similar classes implemented in the future.